cmd/dep/testdata/harness_tests/ensure/empty/case1/final/lock.toml

```diff
@@ -0,0 +1,7 @@
+memo = "63510efb9632ec69c1164ce396d7ebea4ad3884b4fa508373da17226d5a39739"
+
+[[projects]]
+  name = "github.com/sdboyer/deptest"
```

Comment:
Okay, it's not exposed but seems like it wouldn't be hard to add a flag to control the auto-indentation that it's currently doing. I'll submit a PR to `go-toml` for that.